tripy/tripy/frontend/trace/ops/utils.py

```diff
@@ -113,6 +113,29 @@ def never_return_shape(self, inputs):
         return Result.ok([])
 
 
+##
+## Inferring shape lengths (helpers)
+##
+
+
+def get_op_input_shape(input: "TraceTensor") -> Sequence[int]:
```

Comment:
Why not just `get_shape`?

Reply:
Changed it to `get_trace_shape` to avoid confusion with other functions with names of the form `get_shape*`